### PR TITLE
COMMERCE-12362 Test Automation for COMMERCE-11664 and COMMERCE-11662

### PR DESCRIPTION
**Pull request by pietrobocale**
https://liferay.atlassian.net/browse/COMMERCE-12362

### CODE REVIEW

**Comment by pietrobocale** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceUnitsOfMeasure.testcase [hunk 1]
```diff
@@ -641,6 +1411,341 @@ definition {
...
+			AssertElementPresent(
+				key_cartItemSku = "MIN55861",
+				key_optionValue = "0.5",
+				locator1 = "CommerceAccelerators#MINI_CART_ITEM_QUANTITY_SELECTOR_INPUT_FIELD_OPTION_VALUE");
+		}
+	}
+
+	@description = "This is a test for COMMERCE-12381 | Verify that Price On Application works correctly on the minicart with UOM"
+	@priority = 5
+	test CanPriceOnApplicationConfigurationWorkCorrectlyOnMinicart {
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceUnitsOfMeasure.testcase:1600]:
```suggestion
	test CanPriceOnApplicationWithUOMBeShownInMiniCart {
```

**Review by lorenzolapescara — CHANGES_REQUESTED** [review]
@pietrobocale please have a look at the comments I left.
We also agreed to split the Order Quantity Configuration tests into smaller tests due to complexity.

**Comment by pietrobocale** [comment]
ci:test:sf

**Comment by pietrobocale** [comment]
ci:test:commerce-functional

**Comment by pietrobocale** [comment]
New PR:
https://github.com/liferay-commerce/liferay-portal/pull/4305

**Pull request closed without merging**